graphql/http.go

```diff
@@ -0,0 +1,97 @@
+package graphql
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/samsarahq/thunder/batch"
+	"github.com/samsarahq/thunder/reactive"
+)
+
+func CreateHTTPHandler(schema *Schema, middlewares ...MiddlewareFunc) http.Handler {
```

Comment:
is it time to start adding tests for the servers interfaces yet

Reply:
yeah let's do it. (for this one)

Reply:
I added some tests. Let's open source the snapshotter tool, because I want to use it.